pelita/maze_generator.py

```diff
@@ -202,7 +201,7 @@ def _add_wall(maze, ngaps, vertical, rng):
             _add_wall(sub_maze, max(1, ngaps // 2), not vertical, rng=rng)
 
 
-def maze_to_graph(maze):
+def walls_to_graph(maze):
```

Comment:
Maybe we could make it a bit more explicit (in name, docs and/or type annotation) which functions take a numpy maze and which take a list of wall positions?

Reply:
yes, yes, definitely. This is at the moment a moving target, we are changing and adapting the code a lot. It is really a big WIP. I'll ask here for review as soon as the code has stabilized a bit. There are all sorts of horrors in it at the moment :-)

Reply:
Yeah, I thought so. I wish Github had a feature that made these comments private until the review is ready. I wrote it so *I* won’t forget. :)